src/lib.rs

```diff
@@ -711,6 +769,14 @@ fn parse_field(
                 ));
             }
 
+            // Ensure padding fields remain inaccesible
+            if ignore && access.filter(|mode| *mode != AccessMode::None).is_some() {
+                return Err(syn::Error::new(
```

Comment:
`access.is_some_and(|a| a != AccessMode::None)` might be a little bit more clear than filter.

Reply:
There are so many methods on `Option` and `Result`, I always confuse them :D